### PR TITLE
Recipe update for pulling vscode 1.90.1

### PR DESCRIPTION
**Pull request by aws-pangestu**
Checklist
* [x ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by aws-pangestu** [comment]
@conda-forge-admin, please rerender

**Comment by aws-pangestu** [comment]
@conda-forge-admin, please rerender

**Review by navinsoni — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,8 +17,9 @@ build:
 
 requirements:
   build:
+    - compilers
```
Comment (navinsoni) [inline recipe/meta.yaml:20]: Can you please remove compiler('c') and compiler('cxx')

**Review by navinsoni — APPROVED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version|replace("-", "_") }}
 
 source:
   # github repo set-up is in progress, once ready we will use the github release url here instead of the s3 url
-  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor1.0.1.tar.gz
-  sha256: 8430cbc38b6a54814f970f2d23c8f5aa1fb7d3c26511c18d5f250d91f110230d
+  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor1.1.0.tar.gz
```
Comment (arkaprava08) [inline recipe/meta.yaml:10]: Can't we use the github's tar file instead of custom S3?

**Pull request closed without merging**